package.json

```diff
@@ -2,10 +2,10 @@
   "name": "holo-hylo-dna",
   "version": "0.0.5",
   "scripts": {
-    "start": "holochain -c ./conductor-config.toml",
-    "start:agent1": "holochain -c ./conductor-config-agent1.toml",
-    "start:agent2": "holochain -c ./conductor-config-agent2.toml",
-    "start:agent3": "holochain -c ./conductor-config-agent3.toml",
+    "start": "holochain -c ./conductor-configs/conductor-config.toml",
```

Comment:
This is awesome. Was wanting that. I think there is a case to keep`conductor-config.toml` in root (and .gitignore'd as it is) and that be the default for `yarn start` such that we can maintain a local config for an agent which is customized.

Reply:
Ah, I see now that I have it checked-out I see that `conductor-config.toml` of course will still be .gitignored while in the `conductor-configs` directory. For discussion--depending on how running a local agent with dnas would settle over time, it might still make the most sense to have the default config be at the root and look at the other conductor-configs as "test configs" (maybe even naming the directory `testing-conductor-configs` or something). Nothing is settled on this stuff I think so either way, but this is my hunch is that having a default conductor-config at root might be most elegant...